slsDetectorServers/gotthard2DetectorServer/slsDetectorFunctionList.c

```diff
@@ -2578,6 +2578,23 @@ int getVeto() {
             CONFIG_VETO_ENBL_OFST);
 }
 
+void setVetoStream(int value) {
```

Comment:
Does setVetoStream only consider the 3gbe interface?

Reply:
In the detector server yes, because the setNumberofUDPInterfaces already does this for 10gbe and does a whole lot of other things related to it cuz of zmq etc.

Reply:
I see